### PR TITLE
Add scope to client credentials token

### PR DESCRIPTION
**Pull request by vahpetr**
Added scope field to token. The field is required for validation access policy on the server side

### CODE REVIEW

**Comment by vahpetr** [comment]
up

File: src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs [hunk 1]
```diff
@@ -479,6 +479,7 @@ private async Task<IActionResult> ExchangeClientCredentialsGrantType(OpenIddictR
             }
 
             var principal = new ClaimsPrincipal(identity);
+            principal.SetScopes(request.GetScopes());
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs:482]: @kevinchalet LGTM but can you confirm, thanks.

**Review by kevinchalet — APPROVED** [review]
Thanks for your PR, @vahpetr!

**Comment by kevinchalet** [comment]
PR merged, thanks again for your contribution 👏🏻

**Pull request merged**